cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/types.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+interface IArtifact {
+  name: string;
```

Comment:
nit: Should this have version and scope?

Reply:
This is what I'm not 100% sure, while yes, the backend information should have those, it is not being used in this list view. Do we want to still enforce that on the interface?

Reply:
Hhmm I see. Then I think the name alone in the type would be ok. When we start to use the other information we can probably add it to this. 👍